### PR TITLE
Add more comprehensive amp-story example

### PR DESCRIPTION
**Pull request by newmuis**

### CODE REVIEW

**Comment by newmuis** [comment]
@calebcordry Do you know if it's possible to get ads to work here?  Would we have to set up something custom in the app.js server to randomly select an ad (in Node, this time)?

**Comment by calebcordry** [comment]
We can make it work. There are a couple of options, the easiest would be to just point it at the existing ampbyexample.com endpoint (it already returns random ad) . Are you ok with that?

**Comment by calebcordry** [comment]
here is the endpoint: https://ampbyexample.com/json/amp-story-auto-ads/

**Comment by newmuis** [comment]
Yep, that should be fine.  I'll update the example.

**Comment by rsimha** [comment]
@newmuis Perhaps make this as a visual diff test? (Can be done separately.)

File: examples/amp-story/json/amp-story-auto-ads-0.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (calebcordry) [inline examples/amp-story/json/amp-story-auto-ads-0.json]: We aren't actually using any of these json files, since we are using the ABE backend.

**Comment by newmuis** [comment]
We want to have visual diff tests, but I think we would benefit more from having them be more targeted.  I was also never able to figure out why our loading CSS classes weren't working in #11682, but it seemed (at the time) related to the experiment.

File: examples/amp-story/json/amp-story-auto-ads-0.json (hunk 1 above)
Comment (newmuis) [reply examples/amp-story/json/amp-story-auto-ads-0.json]: Deleted.

**Pull request merged**